_plugins/build.rb

```diff
@@ -0,0 +1,12 @@
+Jekyll::Hooks.register :site, :pre_render do |site|
```

Comment:
Unused block argument - `site`. You can omit the argument if you don't care about it.